lighthouse-core/runner.js

```diff
@@ -254,7 +254,12 @@ class Runner {
 
       Sentry.captureException(err, {tags: {audit: audit.meta.name}, level: 'error'});
       // Non-fatal error become error audit result.
-      return Audit.generateErrorAuditResult(audit, 'Audit error: ' + err.message);
+      let debugString = `Audit error: ${err.message}`;
+      if (err.friendlyMessage) {
```

Comment:
style nite: let's do a ternary for this `const debugString` assignment instead.

Reply:
done